src/components/renderer/AutocompleteDropdown.tsx

```diff
@@ -114,15 +114,16 @@ function AutocompleteDropdown<T>({
         }
```

Comment:
I think we probably only want the `onSelectOption` to happen if that option is not already selected.